lib/ai_bot/playground.rb

```diff
@@ -273,6 +273,12 @@ def reply_to(post)
           reply_post.post_custom_prompt.update!(custom_prompt: prompt)
         end
 
+        # since we are skipping validations and jobs we
+        # may need to fix participant count
+        if reply_post.topic.private_message? && reply_post.topic.participant_count < 2
+          reply_post.topic.update(participant_count: 2)
```

Comment:
```suggestion
          reply_post.topic.update!(participant_count: 2)
```

We generally prefer this

Reply:
worried about exploding the bot on this... but I guess its ok.